specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,641 @@
+          "x-nullable": false
+        },
+        "trainingDocuments": {
+          "description": "List of documents used to train the model and the\r\ntrain operation error reported by each.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormDocumentReport"
+          }
+        },
+        "errors": {
+          "description": "Errors returned during the training operation.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormOperationError"
+          }
+        }
+      }
+    },
+    "KeysResult": {
+      "description": "Result of an operation to get\r\nthe keys extracted by a model.",
+      "type": "object",
+      "properties": {
+        "clusters": {
+          "description": "Object mapping ClusterIds to Key lists.",
+          "type": "object",
+          "additionalProperties": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": { "type": "string" }
+          }
+        }
+      }
+    },
+    "ModelResult": {
+      "description": "Result of a model status query operation.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Get or set model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [ "created", "ready", "invalid" ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Get or set the created date time of the model.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Get or set the model last updated datetime.",
+          "type": "string"
+        }
+      }
+    },
+    "ModelsResult": {
+      "description": "Result of query operation to fetch multiple models.",
+      "type": "object",
+      "properties": {
+        "models": {
+          "description": "Collection of models.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelResult"
+          }
+        }
+      }
+    },
+    "FormDocumentReport": {
+      "type": "object",
+      "properties": {
+        "documentName": {
+          "description": "Reference to the data that the report is for.",
+          "type": "string"
+        },
+        "pages": {
+          "format": "int32",
+          "description": "Total number of pages trained on.",
+          "type": "integer"
+        },
+        "errors": {
+          "description": "List of errors per page.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "status": {
```

Comment:
@ramparab shouldn't status be readOnly?

Reply:
In this case, the status relates to the training data and training operation using that data. Hope this clarifies.

Reply:
Will the value ever be sent from the client to the service? Can the client change the value on the service?

Reply:
Ah.. no. Client will never be able to send these values to service. These are results sent by /Train call to service.